README.md

```diff
@@ -29,44 +30,93 @@ Note that `words` should be automatically installed by `pip` when you follow
 the installation instructions for this package.
 
 ## Directory Structure
+
 * `analysis_config/` contains a configuration file which can be used for analysis of cost, latency, and performance metrics for one set of hyperparameters for a particular study. This config is used in the script scripts/anaylsis_for_one_study.py
 * `config/` contains configuration files which specify hyperparamter settings for running experiments.
 * `hypothesis_config/` contains a configuration file which specifies filtering criterias for generating figures for various hypotheses.
-* `notebooks/ ` has a Jupyter notebook for generating figures that are used in the research paper
+* `notebooks/` has a Jupyter notebook for generating figures that are used in the research paper
 * `scripts/` contains Python scripts for running experiments, post-processing the results, and analysis of results
 * `setlexsem/` is the module which has all the important functions and utils for analysis, experimentation, generation of data, samplers.
-    * `analyze` contains code for error_analysis of post-processed results, visualizaiton code and utils neeeded for generating figures for hypothesis.
-    * `experiment` contains code for invoking LLMs and running experiments for a particular hypothesis/study.
-    * `generate` contains code for generating data, sample synthetic sets, prompts and utils needed for data generation.
-    * `prepare` contains helper functions for partitioning words according to their frequencies.
+  * `analyze` contains code for error_analysis of post-processed results, visualizaiton code and utils neeeded for generating figures for hypothesis.
+  * `experiment` contains code for invoking LLMs and running experiments for a particular hypothesis/study.
+  * `generate` contains code for generating data, sample synthetic sets, prompts and utils needed for data generation.
+  * `prepare` contains helper functions for partitioning words according to their frequencies.
+
+## Data Generation
+
+### Numbers and Words
+
+To generate your own data, you can run the following:
```

Comment:
Does this generate the prompts? Or just the samples?

Reply:
only samples, will add that in the revision